### PR TITLE
#90 - Added Content interface to represent value stored in Storage

### PR DESCRIPTION
**Pull request by olegmoz**
Closes issue #90 
Added Content interface to represent value stored in Storage.

### CODE REVIEW

**Comment by paulodamaso** [comment]
@g4s8 please assign me

**Comment by HDouss** [comment]
@paulodamaso Since no answer from @g4s8, I will start the review here.

File: src/main/java/com/artipie/asto/fs/RxFileContent.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        this.file = file;
+    }
+
+    @Override
+    public Publisher<ByteBuffer> bytes() {
+        return this.file.flow();
+    }
+
+    @Override
+    public Optional<Long> size() {
```
[the remark this answers is not in the document]
Comment (olegmoz) [reply src/main/java/com/artipie/asto/fs/RxFileContent.java]: @HDouss yeah, good point! Added `size()` method to `RxFile` and it's usage in 'RxFileContent'

**Comment by HDouss** [comment]
@g4s8 Review finished. Please review or/and merge.

**Comment by g4s8** [comment]
@Sammers21 can you please take a look?

File: src/main/java/com/artipie/asto/Content.java [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+import java.nio.ByteBuffer;
+import java.util.Optional;
+import org.reactivestreams.Publisher;
+
+/**
+ * Content that can be stored in {@link Storage}.
+ *
+ * @since 0.15
+ */
+public interface Content {
```
Comment (Sammers21) [inline src/main/java/com/artipie/asto/Content.java]: Can we extend this interface with `Publisher<ByteBuffer>`?

Comment (Sammers21) [reply src/main/java/com/artipie/asto/Content.java]: And remove `#bytes` method

Comment (olegmoz) [reply src/main/java/com/artipie/asto/Content.java]: @Sammers21 made the changes

**Comment by olegmoz** [comment]
@Sammers21 @HDouss @g4s8 changed Content to extend Publisher, please review

**Comment by 0crat** [comment]
Job audit: performer @hdouss[/z](https://www.0crat.com/u/hdouss)[/z](https://www.0crat.com/u/hdouss) didn't make CR comments

**Comment by 0crat** [comment]
Job was finished in 23 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Pull request merged**